README.md

```diff
@@ -70,7 +70,7 @@ To support tight fitting volumes for a variety of datasets from regularly divide
 
 _TODO: include a screenshot of each bounding volume type._
 
-A tile usually references a _model_ or set of _models_, e.g., 3D buildings.  These models may be batched together into essentially a single model to reduce client-side load time and WebGL draw call overhead.
+A tile references a _feature_ or set of _features_, such as 3D models representing buildings or trees, points in a point cloud, or polygons, polylines, and points in a vector dataset.  These features may be batched together into essentially a single feature to reduce client-side load time and WebGL draw call overhead.
```

Comment:
It makes as much sense as it does to call it a model.  Geospatial folks think of a polygon as a feature, if we combine two polygons into one polygon, that batched polygon would also be a feature.  It is not as native to us as graphics developers, but I think it is fine.